polyfill/test/Instant/prototype/toZonedDateTime/plain-custom-timezone.js

```diff
@@ -7,7 +7,10 @@ includes: [compareArray.js]
 ---*/
 
 const actual = [];
-const expected = [];
+const expected = [
+  "has timeZone.timeZone", // ToTemporalTimeZone in Instant.prototype.toZonedDateTime
```

Comment:
These don't occur in the `from` calls at all, but both occur internally in `Temporal.Instant.prototype.toZonedDateTime()`. It's called once on the argument at the beginning of the method, and once when constructing the new instance of ZonedDateTime. I agree this is not ideal, and I just wondered if it might be good to change CreateZonedDateTime to _not_ call ToTemporalZonedDateTime, but instead move that out to all the callers or remove it if, as in this case, the time zone is already guaranteed to be a valid value. What do you think?

Reply:
We discussed this out of band and decided that this change would be within the mandate of this consensus (before the change requested in this issue, there were zero observable Has operations, now there are two, and ideally there is only one). I'll make this change, and then land the pull request.

Reply:
Of course, as soon as I go to make the change I realize that it is already this way in the spec text, but we just need to make sure it is reflected in the polyfill.

Reply:
I've fixed the bug in the polyfill and will go ahead and merge this today, so we can keep the normative changes moving. I'm happy to receive comments on the polyfill change after the fact, and can fix any mistakes up anytime, since it is not normative.